assets/src/edit-story/components/panels/alignment/alignment.js

```diff
@@ -156,7 +170,6 @@ function ElementAlignmentPanel({ selectedElements, pushUpdate }) {
     [updatedSelectedElementsWithFrame, setUpdatedSelectedElementsWithFrame]
   );
 
-  const isAlignEnabled = !isFill && selectedElements.length > 1;
```

Comment:
So now you can align `isFill` elements? 🤔

Reply:
Yes, we need to disable for isFill element. Thank you